### DISCUSSION TITLE
ImageMagick Command Not Recognized on Azure Windows Server 2019 VM

### DISCUSSION THREAD

**Discussion by mignetically** [ImageMagick/ImageMagick#7356] [Help]
### ImageMagick version

7.1.1-33-Q16-HDRI-x64

### Operating system, version and so on

Windows Server 2019 Datacenter

### Description

I am experiencing an issue where the ImageMagick command is not recognized in the Command Prompt on my Azure Windows Server 2019 VM. Despite following the installation steps, the command magick is not recognized, and I cannot get ImageMagick to run properly.

Environment:
Operating System: Windows Server 2019 Datacenter
VM Size: Standard NV8as v4 (8 vCPUs, 28 GiB memory)
Installed Version: ImageMagick-7.1.1-33-Q16-HDRI-x64-dll.exe
RAM: 32 GB
Steps Taken:
Installation: Downloaded and installed ImageMagick using the installer ImageMagick-7.1.1-33-Q16-HDRI-x64-dll.exe.
Path Variable: Verified that the installation path C:\Program Files\ImageMagick-7.1.1-Q16-HDRI is included in the system PATH variable.
Direct Execution:
Attempted to run magick -version in Command Prompt, but received the error: magick' is not recognized as an internal or external command, operable program or batch file.
Reinstallation: Tried reinstalling ImageMagick multiple times and ensured to select the option to add it to the system PATH during installation.
Verification: Checked that magick.exe is present in the installation directory.
Environment Variables:
Opened Command Prompt and typed echo %PATH% to verify that the path to ImageMagick is included.
Full Path Execution:
Tried running the command using the full path C:\Program Files\ImageMagick-7.1.1-Q16-HDRI\magick.exe -version, but it still does not execute.
Permissions:
Checked file permissions to ensure magick.exe can be executed.
Error Messages:
'magick' is not recognized as an internal or external command, operable program or batch file.
Questions:
Are there any additional steps required to configure ImageMagick on a Windows Server 2019 VM that I might be missing?
Could there be any compatibility issues with the VM size or configuration that could prevent ImageMagick from running properly?
Are there specific permissions or security settings on Windows Server 2019 that I need to adjust?
Is there a known issue with the version of ImageMagick I am using on this specific environment?
Any help or guidance on resolving this issue would be greatly appreciated. Thank you!

**Comment by snibgo** [comment]
I suggest you `cd` to the directory that contains `magick.exe`. Then, what is the text output from `dir`?

**Comment by dlemstra** [comment]
I just checked if I could reproduce your issue but I can install and run ImageMagick on an Azure Windows Server 2019 VM without any issues.

![issue 7356](https://github.com/ImageMagick/ImageMagick/assets/10426229/ad22e0e6-69de-4324-89f2-2c845c266ad0)
